--- agocontrol-2014-02-11.org/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/CMakeLists.txt	2014-02-22 17:22:13.652256974 +0100
@@ -0,0 +1,37 @@
+cmake_minimum_required (VERSION 2.6)
+project (agocontrol)
+
+# The version number.
+set (CMAKE_AGOCONTROL_VERSION 0.1)
+
+# some configurable values
+set (BINDIR /opt/agocontrol/bin)
+set (ETCDIR /etc)
+set (LIBDIR /usr/lib)
+set (CONFDIR ${ETCDIR}/opt/agocontrol)
+set (INCDIR /usr/include/agocontrol)
+set (DATADIR /var/opt/agocontrol)
+set (LOCALSTATEDIR /var/opt/agocontrol)
+set (HTMLDIR /opt/agocontrol/html)
+
+
+# configure a header file to pass some of the CMake settings
+# to the source code
+configure_file (
+    "${PROJECT_SOURCE_DIR}/version.h.in"
+    "${PROJECT_SOURCE_DIR}/version.h"
+    @ONLY
+)
+ 
+# add the binary tree to the search path for include files
+# so that we will find version.h
+include_directories("${PROJECT_BINARY_DIR}")
+
+include_directories ("${PROJECT_SOURCE_DIR}/shared")
+
+add_subdirectory (shared) 
+
+add_subdirectory (core) 
+#add_subdirectory (devices) 
+add_subdirectory (conf) 
+add_subdirectory (scripts) 
--- agocontrol-2014-02-11.org/conf/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/conf/CMakeLists.txt	2014-02-22 17:17:15.938929372 +0100
@@ -0,0 +1,19 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB INIT_FILES *.in)
+
+foreach (infile ${INIT_FILES})
+
+    string(REGEX REPLACE ".in$" "" outfile ${infile})
+
+    configure_file(
+        "${infile}"
+        "${outfile}"
+        @ONLY
+    )
+
+endforeach (infile)
+
+add_subdirectory (systemd)
+add_subdirectory (conf.d)
+add_subdirectory (sysvinit)
--- agocontrol-2014-02-11.org/conf/conf.d/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/conf/conf.d/CMakeLists.txt	2014-02-22 17:16:47.678423811 +0100
@@ -0,0 +1,15 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB CONFD_FILES *.in)
+
+foreach (infile ${CONFD_FILES})
+
+    string(REGEX REPLACE ".in$" "" outfile ${infile})
+
+    configure_file(
+        "${infile}"
+        "${outfile}"
+        @ONLY
+    )
+
+endforeach (infile)
--- agocontrol-2014-02-11.org/conf/systemd/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/conf/systemd/CMakeLists.txt	2014-02-22 17:08:24.805432896 +0100
@@ -0,0 +1,15 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB SYSTEMD_FILES *.in)
+
+foreach (infile ${SYSTEMD_FILES})
+
+    string(REGEX REPLACE ".in$" "" outfile ${infile})
+
+    configure_file(
+        "${infile}"
+        "${outfile}"
+        @ONLY
+    )
+
+endforeach (infile)
--- agocontrol-2014-02-11.org/conf/sysvinit/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/conf/sysvinit/CMakeLists.txt	2014-02-22 17:15:48.889372214 +0100
@@ -0,0 +1,7 @@
+cmake_minimum_required (VERSION 2.6)
+
+configure_file(
+    "${CMAKE_CURRENT_SOURCE_DIR}/agocontrol.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/agocontrol"
+    @ONLY
+)
--- agocontrol-2014-02-11.org/core/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/CMakeLists.txt	2014-02-22 16:45:59.105362244 +0100
@@ -0,0 +1,23 @@
+cmake_minimum_required (VERSION 2.6)
+
+# configure a header file to pass some of the CMake settings
+# to the source code
+configure_file (
+    "${CMAKE_CURRENT_SOURCE_DIR}/agocontrol-init-script.sh.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/agocontrol-init-script.sh"
+    @ONLY
+)
+
+install (FILES agodrain.py DESTINATION bin)
+install (FILES myavahi.py DESTINATION bin)
+install (FILES agoctrl.sh DESTINATION bin)
+
+add_subdirectory (resolver)
+add_subdirectory (rpc)
+add_subdirectory (messagesend)
+add_subdirectory (timer)
+add_subdirectory (scenario)
+add_subdirectory (event)
+add_subdirectory (datalogger)
+add_subdirectory (lua)
+add_subdirectory (security)
--- agocontrol-2014-02-11.org/core/datalogger/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/datalogger/CMakeLists.txt	2014-02-22 13:57:11.041958763 +0100
@@ -0,0 +1,12 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB_RECURSE AGODATALOGGER_SOURCE_FILES *.cpp)
+set (DATALOGGER_LIBRARIES
+    agoclient
+    sqlite3
+    boost_date_time
+)
+
+# add the executable
+add_executable (agodatalogger ${AGODATALOGGER_SOURCE_FILES})
+target_link_libraries (agodatalogger ${DATALOGGER_LIBRARIES})
--- agocontrol-2014-02-11.org/core/event/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/event/CMakeLists.txt	2014-02-22 13:57:32.806346028 +0100
@@ -0,0 +1,11 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB_RECURSE AGOEVENT_SOURCE_FILES *.cpp)
+set (EVENT_LIBRARIES
+    agoclient
+    boost_system
+)
+
+# add the executable
+add_executable (agoevent ${AGOEVENT_SOURCE_FILES})
+target_link_libraries (agoevent ${EVENT_LIBRARIES})
--- agocontrol-2014-02-11.org/core/lua/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/lua/CMakeLists.txt	2014-02-22 16:45:43.785087238 +0100
@@ -0,0 +1,19 @@
+cmake_minimum_required (VERSION 2.6)
+
+configure_file (
+    "${CMAKE_CURRENT_SOURCE_DIR}/command.lua.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/command.lua"
+    @ONLY
+)
+
+
+file (GLOB_RECURSE AGOLUA_SOURCE_FILES *.cpp)
+set (LUA_LIBRARIES
+    agoclient
+    lua5.2
+)
+
+# add the executable
+add_executable (agolua ${AGOLUA_SOURCE_FILES})
+target_link_libraries (agolua ${LUA_LIBRARIES})
+
--- agocontrol-2014-02-11.org/core/messagesend/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/messagesend/CMakeLists.txt	2014-02-22 13:56:48.265553466 +0100
@@ -0,0 +1,9 @@
+cmake_minimum_required (VERSION 2.6)
+
+set (MESSAGESEND_LIBRARIES
+    agoclient
+)
+
+# add the executable
+add_executable (messagesend messagesend.cpp)
+target_link_libraries (messagesend ${MESSAGESEND_LIBRARIES})
--- agocontrol-2014-02-11.org/core/resolver/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/resolver/CMakeLists.txt	2014-02-22 13:56:59.985762026 +0100
@@ -0,0 +1,14 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB_RECURSE AGORESOLVER_SOURCE_FILES *.cpp)
+
+set (RESOLVER_LIBRARIES
+    yaml-cpp
+    jsoncpp
+    sqlite3
+    agoclient
+)
+
+# add the executable
+add_executable (agoresolver ${AGORESOLVER_SOURCE_FILES})
+target_link_libraries (agoresolver ${RESOLVER_LIBRARIES})
--- agocontrol-2014-02-11.org/core/rpc/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/rpc/CMakeLists.txt	2014-02-22 16:45:32.096877432 +0100
@@ -0,0 +1,26 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB_RECURSE AGORPC_SOURCE_FILES *.cpp)
+
+set (RPC_LIBRARIES
+    agoclient
+)
+
+include_directories(
+    /usr/include/jsoncpp
+)
+
+# add the executable
+#add_executable (agorpc ${AGORPC_SOURCE_FILES})
+add_executable (agorpc agorpc.cpp mongoose.c)
+target_link_libraries (agorpc ${RPC_LIBRARIES})
+
+add_executable (get_table_lang.cgi get_table_lang.cpp)
+
+# configure a header file to pass some of the CMake settings
+# to the source code
+configure_file (
+    "${CMAKE_CURRENT_SOURCE_DIR}/activate.cgi.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/activate.cgi"
+    @ONLY
+)
--- agocontrol-2014-02-11.org/core/scenario/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/scenario/CMakeLists.txt	2014-02-22 14:03:37.073002132 +0100
@@ -0,0 +1,9 @@
+cmake_minimum_required (VERSION 2.6)
+
+set (SCENARIO_LIBRARIES
+    agoclient
+)
+
+# add the executable
+add_executable (agoscenario agoscenario.cpp)
+target_link_libraries (agoscenario ${SCENARIO_LIBRARIES})
--- agocontrol-2014-02-11.org/core/security/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/security/CMakeLists.txt	2014-02-22 14:04:37.738116113 +0100
@@ -0,0 +1,9 @@
+cmake_minimum_required (VERSION 2.6)
+
+set (SECURITY_LIBRARIES
+    agoclient
+)
+
+# add the executable
+add_executable (agosecurity agosecurity.cpp)
+target_link_libraries (agosecurity ${SECURITY_LIBRARIES})
--- agocontrol-2014-02-11.org/core/timer/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/core/timer/CMakeLists.txt	2014-02-22 14:00:48.917911968 +0100
@@ -0,0 +1,12 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB_RECURSE AGOTIMER_SOURCE_FILES *.cpp)
+set (TIMER_LIBRARIES
+    agoclient
+    boost_system
+    hdate
+)
+
+# add the executable
+add_executable (agotimer ${AGOTIMER_SOURCE_FILES})
+target_link_libraries (agotimer ${TIMER_LIBRARIES})
--- agocontrol-2014-02-11.org/scripts/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/scripts/CMakeLists.txt	2014-02-22 17:18:22.876126943 +0100
@@ -0,0 +1,15 @@
+cmake_minimum_required (VERSION 2.6)
+
+file (GLOB SCRIPT_FILES *.in)
+
+foreach (infile ${SCRIPT_FILES})
+
+    string(REGEX REPLACE ".in$" "" outfile ${infile})
+
+    configure_file(
+        "${infile}"
+        "${outfile}"
+        @ONLY
+    )
+
+endforeach (infile)
--- agocontrol-2014-02-11.org/shared/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/shared/CMakeLists.txt	2014-02-22 16:45:11.004498827 +0100
@@ -0,0 +1,35 @@
+cmake_minimum_required (VERSION 2.6)
+
+# configure a header file to pass some of the CMake settings
+# to the source code
+configure_file (
+    "${CMAKE_CURRENT_SOURCE_DIR}/agoclient.h.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/agoclient.h"
+    @ONLY
+)
+
+configure_file (
+    "${CMAKE_CURRENT_SOURCE_DIR}/agoclient.py.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/agoclient.py"
+    @ONLY
+)
+
+file(GLOB_RECURSE LIBAGOCLIENT_SOURCE_FILES *.cpp)
+
+add_library (agoclient SHARED ${LIBAGOCLIENT_SOURCE_FILES})
+set_target_properties (agoclient PROPERTIES VERSION 1.0 SOVERSION 1)
+
+set (AGOCLIENT_LIBRARIES
+    qpidmessaging
+    qpidtypes
+    uuid
+    jsoncpp
+)
+
+target_link_libraries (agoclient ${AGOCLIENT_LIBRARIES})
+
+# Install instructions for this target
+install (TARGETS agoclient LIBRARY DESTINATION lib)
+
+install (FILES agoclient.h DESTINATION include)
+install (FILES agoclient.py DESTINATION lib)
--- agocontrol-2014-02-11.org/version.h.in	1970-01-01 01:00:00.000000000 +0100
+++ agocontrol-2014-02-11/version.h.in	2014-02-22 12:26:19.573398192 +0100
@@ -0,0 +1 @@
+#define AGOCONTROL_VERSION @CMAKE_AGOCONTROL_VERSION@
