--- agocontrol-2014-02-11.org/core/rpc/Makefile	2014-02-11 14:01:50.000000000 +0100
+++ agocontrol-2014-02-11/core/rpc/Makefile	2014-02-22 03:05:29.909803612 +0100
@@ -5,15 +5,15 @@
 %.o : %.cpp
 	$(CXX) -c $(CFLAGS) $(INCLUDES) -o $@ $<
 
-%.html : %.in.html
-	./mergetranslation.sh $< $@
+#%.html : %.in.html
+#	./mergetranslation.sh $< $@
 
 all: agorpc html cgi
 
 agorpc: agorpc.o mongoose.o
-	$(LD) -o $@ $(LDFLAGS) $^ -lqpidmessaging -lqpidtypes -luuid -ljsoncpp -lagoclient
+	$(LD) -o $@ $(LDFLAGS) $^ -lqpidmessaging -lqpidtypes -luuid -ljson_cpp -lagoclient -ldl -lqpidcommon -lqpidclient -lboost_system -lboost_program_options -lrt -lstdc++ -lgcc_s -lpthread -lc
 
-html: $(patsubst %.in.html,%.html,$(wildcard html/templates/*.in.html) $(wildcard html/templates/details/*.in.html) $(wildcard html/templates/devices/*.in.html) $(wildcard html/templates/configuration/*.in.html) $(wildcard html/templates/navigation/*.in.html))
+#html: $(patsubst %.in.html,%.html,$(wildcard html/templates/*.in.html) $(wildcard html/templates/details/*.in.html) $(wildcard html/templates/devices/*.in.html) $(wildcard html/templates/configuration/*.in.html) $(wildcard html/templates/navigation/*.in.html))
 
 cgi:  get_table_lang.cpp
 	$(CXX) $(CFLAGS) -o html/cgi-bin/get_table_lang.cgi get_table_lang.cpp
