diff -uNr agocontrol-2014-02-11.org/core/resolver/Makefile agocontrol-2014-02-11/core/resolver/Makefile
--- agocontrol-2014-02-11.org/core/resolver/Makefile	2014-02-11 14:01:50.000000000 +0100
+++ agocontrol-2014-02-11/core/resolver/Makefile	2014-02-22 02:58:56.414819120 +0100
@@ -6,7 +6,7 @@
 all: agoresolver
 
 agoresolver: agoresolver.o schema.o inventory.o
-	$(LD) -o $@ $(LDFLAGS) $^ -lqpidmessaging -lqpidtypes -luuid -lyaml-cpp  -lsqlite3 -lagoclient
+	$(LD) -o $@ $(LDFLAGS) $^ -lqpidmessaging -lqpidcommon -lqpidclient -lqpidtypes -luuid -lyaml-cpp  -lsqlite3 -lagoclient -lrt -lstdc++ -lpthread -lc -lboost_system -lboost_program_options -ljson_cpp -lboost_filesystem
 
 test:
 	$(CXX) $(CFLAGS) $(INCLUDES) -DINVENTORY_TEST -o inventorytest inventory.cpp  -lsqlite3  -lqpidtypes 
