--- agocontrol-2014-02-11.org/Makefile	2014-02-11 14:01:50.000000000 +0100
+++ agocontrol-2014-02-11/Makefile	2014-02-21 18:49:28.458557388 +0100
@@ -3,14 +3,14 @@
 export CC = gcc
 export CXX = g++
 export LD = g++ 
-export LDFLAGS = -L../../shared
-export CFLAGS = -Wall -Wno-format -g -DDEBUG -O2 -pipe
+export LDFLAGS += -L../../shared
+export CFLAGS += -Wall -Wno-format -g -DDEBUG -O2 -pipe
 export INSTALL = install
 export INSTALL_DIR = $(INSTALL) -p -d -o root -g root  -m  755
 export INSTALL_PROGRAM = $(INSTALL) -p    -o root -g root  -m  755
 export INSTALL_FILE    = $(INSTALL) -p    -o root -g root  -m  644
 
-export INCLUDES = -I../../shared
+export INCLUDES += -I../../shared
 
 export BINDIR = /opt/agocontrol/bin
 export ETCDIR = /etc
@@ -38,7 +38,7 @@
 NUMJOBS = $(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
 MAKEFLAGS += -j$(NUMJOBS)
 else
-MAKEFLAGS += -j4
+MAKEFLAGS += -j1
 endif
 export MAKEFLAGS
 
