--- a/conf/Makefile
+++ b/conf/Makefile
@@ -8,7 +8,7 @@ INSTALLDIRS = $(DIRS:%=install-%)
 CLEANDIRS = $(DIRS:%=clean-%)
 
 %.init : %.init.in
-	sed -e 's|@BINDIR|$(BINDIR)|g' $< > $@
+	sed -e 's|@BINDIR@|$(BINDIR)|g' $< > $@
 
 all: $(BUILDDIRS) $(FILES)
 $(DIRS): $(BUILDDIRS)
--- a/conf/agomeloware.init.in
+++ b/conf/agomeloware.init.in
@@ -10,4 +10,4 @@ start on stopped rc RUNLEVEL=[2345]
 stop on runlevel [!2345]
 
 respawn
-exec @BINDIR/agomeloware.py
+exec @BINDIR@/agomeloware.py
--- a/conf/agoowfs.init.in
+++ b/conf/agoowfs.init.in
@@ -10,4 +10,4 @@ start on stopped rc RUNLEVEL=[2345]
 stop on runlevel [!2345]
 
 respawn
-exec @BINDIR/agoowfs.py
+exec @BINDIR@/agoowfs.py
--- a/conf/agoresolver.init.in
+++ b/conf/agoresolver.init.in
@@ -10,4 +10,4 @@ start on stopped rc RUNLEVEL=[2345]
 stop on runlevel [!2345]
 
 respawn
-exec @BINDIR/agoresolver.py
+exec @BINDIR@/agoresolver.py
--- a/conf/agozwave.init.in
+++ b/conf/agozwave.init.in
@@ -10,5 +10,5 @@ start on stopped rc RUNLEVEL=[2345]
 stop on runlevel [!2345]
 
 respawn
-chdir @BINDIR
-exec @BINDIR/agozwave
+chdir @BINDIR@
+exec @BINDIR@/agozwave
--- a/conf/conf.d/Makefile
+++ b/conf/conf.d/Makefile
@@ -2,8 +2,8 @@
 FILES = $(patsubst %.conf.in, %.conf, $(wildcard *.conf.in))
 
 %.conf: %.conf.in
-	sed -e 's|@CONFDIR|$(CONFDIR)|g' \
-		-e 's|@HTMLDIR|$(HTMLDIR)|g' $< > $@
+	sed -e 's|@CONFDIR@|$(CONFDIR)|g' \
+		-e 's|@HTMLDIR@|$(HTMLDIR)|g' $< > $@
 
 all: $(FILES)
 
--- a/conf/conf.d/rpc.conf.in
+++ b/conf/conf.d/rpc.conf.in
@@ -1,5 +1,5 @@
 [rpc]
 ports=8008,8009s
-htdocs=@HTMLDIR
-certificate=@CONFDIR/rpc/rpc_cert.pem
+htdocs=@HTMLDIR@
+certificate=@CONFDIR@/rpc/rpc_cert.pem
 numthreads=50
--- a/conf/systemd/Makefile
+++ b/conf/systemd/Makefile
@@ -2,7 +2,7 @@
 FILES = $(patsubst %.service.in, %.service, $(wildcard *.service.in))
 
 %.service : %.service.in
-	sed -e 's|@BINDIR|$(BINDIR)|g' $< > $@
+	sed -e 's|@BINDIR@|$(BINDIR)|g' $< > $@
 
 all: $(FILES)
 
--- a/conf/systemd/agoapc.service.in
+++ b/conf/systemd/agoapc.service.in
@@ -2,8 +2,8 @@
 Description=ago control APC Device
 
 [Service]
-ExecStart=@BINDIR/agoapc.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoapc.py
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agoasterisk.service.in
+++ b/conf/systemd/agoasterisk.service.in
@@ -2,8 +2,8 @@
 Description=ago control Asterisk PBX interface
 
 [Service]
-ExecStart=@BINDIR/agoasterisk.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoasterisk.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agoblinkm.service.in
+++ b/conf/systemd/agoblinkm.service.in
@@ -2,8 +2,8 @@
 Description=ago control blinkm i2c device
 
 [Service]
-ExecStart=@BINDIR/agoblinkm
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoblinkm
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agodatalogger.service.in
+++ b/conf/systemd/agodatalogger.service.in
@@ -2,8 +2,8 @@
 Description=ago control Data Logger
 
 [Service]
-ExecStart=@BINDIR/agodatalogger
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agodatalogger
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agodmx.service.in
+++ b/conf/systemd/agodmx.service.in
@@ -2,8 +2,8 @@
 Description=ago control dmx device
 
 [Service]
-ExecStart=@BINDIR/agodmx
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agodmx
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agoenigma2.service.in
+++ b/conf/systemd/agoenigma2.service.in
@@ -2,8 +2,8 @@
 Description=ago control Enigma2/Dreambox interface
 
 [Service]
-ExecStart=@BINDIR/agoenigma2.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoenigma2.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agoevent.service.in
+++ b/conf/systemd/agoevent.service.in
@@ -2,8 +2,8 @@
 Description=ago control Event Device
 
 [Service]
-ExecStart=@BINDIR/agoevent
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoevent
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agofirmata.service.in
+++ b/conf/systemd/agofirmata.service.in
@@ -2,8 +2,8 @@
 Description=ago control Arduino Firmata Device
 
 [Service]
-ExecStart=@BINDIR/agofirmata
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agofirmata
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agogc100.service.in
+++ b/conf/systemd/agogc100.service.in
@@ -2,8 +2,8 @@
 Description=ago control gc100 interface
 
 [Service]
-ExecStart=@BINDIR/agogc100.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agogc100.py
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agoi2c.service.in
+++ b/conf/systemd/agoi2c.service.in
@@ -2,8 +2,8 @@
 Description=ago control i2c device
 
 [Service]
-ExecStart=@BINDIR/agoi2c
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoi2c
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agoirtransethernet.service.in
+++ b/conf/systemd/agoirtransethernet.service.in
@@ -2,8 +2,8 @@
 Description=ago control IRTrans Ethernet infrared blaster Device
 
 [Service]
-ExecStart=@BINDIR/agoirtrans_ethernet
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoirtrans_ethernet
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agoiscp.service.in
+++ b/conf/systemd/agoiscp.service.in
@@ -2,8 +2,8 @@
 Description=ago control Onkyo ISCP device
 
 [Service]
-ExecStart=@BINDIR/agoiscp.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoiscp.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agojointspace.service.in
+++ b/conf/systemd/agojointspace.service.in
@@ -2,8 +2,8 @@
 Description=ago control JointSpace Device
 
 [Service]
-ExecStart=@BINDIR/agojointspace.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agojointspace.py
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agoknx.service.in
+++ b/conf/systemd/agoknx.service.in
@@ -2,8 +2,8 @@
 Description=ago control KNX Device
 
 [Service]
-ExecStart=@BINDIR/agoknx
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoknx
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agokwikwai.service.in
+++ b/conf/systemd/agokwikwai.service.in
@@ -2,8 +2,8 @@
 Description=ago control Kwikwai HDMI CEC Bridge Device
 
 [Service]
-ExecStart=@BINDIR/agokwikwai
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agokwikwai
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agomeloware.service.in
+++ b/conf/systemd/agomeloware.service.in
@@ -2,8 +2,8 @@
 Description=ago control Meloware Lightswitchserver Gateway
 
 [Service]
-ExecStart=@BINDIR/agomeloware.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agomeloware.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agoowfs.service.in
+++ b/conf/systemd/agoowfs.service.in
@@ -2,8 +2,8 @@
 Description=ago control 1-Wire Device
 
 [Service]
-ExecStart=@BINDIR/agoowfs.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoowfs.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agoradiothermostat.service.in
+++ b/conf/systemd/agoradiothermostat.service.in
@@ -2,8 +2,8 @@
 Description=ago control radio thermostat device
 
 [Service]
-ExecStart=@BINDIR/agoradiothermostat.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoradiothermostat.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agorain8net.service.in
+++ b/conf/systemd/agorain8net.service.in
@@ -2,8 +2,8 @@
 Description=ago control rain8net Device
 
 [Service]
-ExecStart=@BINDIR/agorain8net
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agorain8net
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=agocontrol
--- a/conf/systemd/agoresolver.service.in
+++ b/conf/systemd/agoresolver.service.in
@@ -2,8 +2,8 @@
 Description=ago control Resolver
 
 [Service]
-ExecStart=@BINDIR/agoresolver
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoresolver
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agorpc.service.in
+++ b/conf/systemd/agorpc.service.in
@@ -2,8 +2,8 @@
 Description=ago control HTTP RPC interface
 
 [Service]
-ExecStart=@BINDIR/agorpc
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agorpc
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agoscenario.service.in
+++ b/conf/systemd/agoscenario.service.in
@@ -2,8 +2,8 @@
 Description=ago control Scenario Device
 
 [Service]
-ExecStart=@BINDIR/agoscenario
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agoscenario
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agosimulator.service.in
+++ b/conf/systemd/agosimulator.service.in
@@ -2,8 +2,8 @@
 Description=ago control Device simulator
 
 [Service]
-ExecStart=@BINDIR/agosimulator.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agosimulator.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agosqueezeboxserver.service.in
+++ b/conf/systemd/agosqueezeboxserver.service.in
@@ -2,8 +2,8 @@
 Description=ago control Squeezebox device
 
 [Service]
-ExecStart=@BINDIR/agosqueezeboxserver.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agosqueezeboxserver.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agotimer.service.in
+++ b/conf/systemd/agotimer.service.in
@@ -2,8 +2,8 @@
 Description=ago control Timer
 
 [Service]
-ExecStart=@BINDIR/agotimer
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agotimer
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agowebcam.service.in
+++ b/conf/systemd/agowebcam.service.in
@@ -2,8 +2,8 @@
 Description=ago control webcam device
 
 [Service]
-ExecStart=@BINDIR/agowebcam
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agowebcam
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agox10.service.in
+++ b/conf/systemd/agox10.service.in
@@ -2,8 +2,8 @@
 Description=ago control X10 Device
 
 [Service]
-ExecStart=@BINDIR/agox10.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agox10.py
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/agozwave.service.in
+++ b/conf/systemd/agozwave.service.in
@@ -2,8 +2,8 @@
 Description=ago control Z-Wave Device
 
 [Service]
-ExecStart=@BINDIR/agozwave
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/agozwave
+WorkingDirectory=@BINDIR@
 StandardOutput=syslog
 StandardError=syslog
 Restart=always
--- a/conf/systemd/raspi1wGPIO.service.in
+++ b/conf/systemd/raspi1wGPIO.service.in
@@ -2,8 +2,8 @@
 Description=ago control raspberry pi 1-wire GPIO Device
 	
 [Service]
-ExecStart=@BINDIR/raspi1wGPIO.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/raspi1wGPIO.py
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=root
--- a/conf/systemd/raspiGPIO.service.in
+++ b/conf/systemd/raspiGPIO.service.in
@@ -2,8 +2,8 @@
 Description=ago control raspberry pi GPIO Device
 	
 [Service]
-ExecStart=@BINDIR/raspiGPIO.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/raspiGPIO.py
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=root
--- a/conf/systemd/raspiMCP3xxxGPIO.service.in
+++ b/conf/systemd/raspiMCP3xxxGPIO.service.in
@@ -2,8 +2,8 @@
 Description=ago raspberry pi MCPxxx GPIO device
 	
 [Service]
-ExecStart=@BINDIR/raspiMCP3xxxGPIO.py
-WorkingDirectory=@BINDIR
+ExecStart=@BINDIR@/raspiMCP3xxxGPIO.py
+WorkingDirectory=@BINDIR@
 Restart=always
 RestartSec=2
 User=root
--- a/conf/sysvinit/Makefile
+++ b/conf/sysvinit/Makefile
@@ -1,7 +1,7 @@
 FILES = agocontrol
 
 agocontrol : agocontrol.in
-	sed -e 's|@BINDIR|$(BINDIR)|g' $< > $@
+	sed -e 's|@BINDIR@|$(BINDIR)|g' $< > $@
 
 all: $(FILES)
 
--- a/conf/sysvinit/agocontrol.in
+++ b/conf/sysvinit/agocontrol.in
@@ -10,7 +10,7 @@
 
 . /lib/lsb/init-functions
 
-BIN_PATH=@BINDIR/
+BIN_PATH=@BINDIR@/
 PID_PATH=/var/run/
 APPS="agorpc agodatalogger agoevent agoresolver agoscenario agotimer agosqueezebox.py agoalert.py agoipx800 agoscheduler.py"
 
--- a/core/Makefile
+++ b/core/Makefile
@@ -12,7 +12,7 @@ $(BUILDDIRS):
 	$(MAKE) -C $(@:build-%=%)
 
 agocontrol-init-script.sh: agocontrol-init-script.sh.in
-	sed -e 's|@BINDIR|$(BINDIR)|g' $< > $@
+	sed -e 's|@BINDIR@|$(BINDIR)|g' $< > $@
 
 install: $(INSTALLDIRS) all
 	$(INSTALL_DIR) $(DESTDIR)$(BINDIR)
--- a/core/agocontrol-init-script.sh.in
+++ b/core/agocontrol-init-script.sh.in
@@ -26,8 +26,8 @@ DESC="ago control core services"
 do_start()
 {
 	echo -n "Starting $DESC:"
-	/usr/bin/screen -d -S agoresolver -m @BINDIR/agoresolver
-	/usr/bin/screen -d -S agoadmin -m @BINDIR/agoadmin.py
+	/usr/bin/screen -d -S agoresolver -m @BINDIR@/agoresolver
+	/usr/bin/screen -d -S agoadmin -m @BINDIR@/agoadmin.py
 	echo "."
 }
 
--- a/core/lua/Makefile
+++ b/core/lua/Makefile
@@ -7,7 +7,7 @@ INCLUDES ?= -I../../shared
 all: agolua command.lua
 
 command.lua : command.lua.in
-	 sed -e 's|@CONFDIR|$(CONFDIR)|g' $< > $@
+	 sed -e 's|@CONFDIR@|$(CONFDIR)|g' $< > $@
 
 agolua: agolua.o
 	$(LD) -o $@ $(LDFLAGS) $^ -lqpidmessaging -lqpidtypes -luuid -lagoclient -llua5.2
--- a/core/lua/command.lua.in
+++ b/core/lua/command.lua.in
@@ -1,4 +1,4 @@
-dofile("@CONFDIR/lua/helper.lua")
+dofile("@CONFDIR@/lua/helper.lua")
 
 for key,value in pairs(content) do print(key,value) end
 -- for key,value in pairs(inventory.devices) do
--- a/core/rpc/Makefile
+++ b/core/rpc/Makefile
@@ -20,7 +20,7 @@ html: $(patsubst %.in.html,%.html,$(wildcard html/templates/*.in.html) $(wildcar
 
 cgi:  get_table_lang.cpp
 	$(CXX) $(CFLAGS) -o html/cgi-bin/get_table_lang.cgi get_table_lang.cpp
-	sed -e 's|@BINDIR|$(BINDIR)|g' activate.cgi.in > html/cgi-bin/activate.cgi
+	sed -e 's|@BINDIR@|$(BINDIR)|g' activate.cgi.in > html/cgi-bin/activate.cgi
 
 clean:
 	rm -f agorpc agorpc.o mongoose.o
--- a/core/rpc/activate.cgi.in
+++ b/core/rpc/activate.cgi.in
@@ -1,6 +1,6 @@
 #!/usr/bin/python
 import sys
-sys.path.append('@BINDIR')
+sys.path.append('@BINDIR@')
 import agoclient
 
 import cgi
--- a/scripts/Makefile
+++ b/scripts/Makefile
@@ -1,5 +1,5 @@
 agososreport.sh: agososreport.sh.in
-	sed -e 's|@CONFDIR|$(CONFDIR)|g' $< > $@
+	sed -e 's|@CONFDIR@|$(CONFDIR)|g' $< > $@
 
 all: agososreport.sh
 
--- a/scripts/agososreport.sh.in
+++ b/scripts/agososreport.sh.in
@@ -3,7 +3,7 @@ TMPFILE=$(mktemp)
 echo creating ago control sosreport, will be saved in $TMPFILE
 (
 echo ==== CONFG ====
-cat @CONFDIR/config.ini | grep -v ^uuid | grep -v ^#
+cat @CONFDIR@/config.ini | grep -v ^uuid | grep -v ^#
 echo ==== PSINF ====
 ps auxw | grep ago | grep -v grep
 echo ==== QPIDD ====
@@ -22,7 +22,7 @@ dpkg -l | grep qpid
 echo ==== LSUSB ====
 lsusb -v
 echo ==== UDEVI ====
-grep "/dev" @CONFDIR/config.ini | cut -d "=" -f 2 | while read file; do echo "$file" $(udevadm info -q path -n "$file"); done
+grep "/dev" @CONFDIR@/config.ini | cut -d "=" -f 2 | while read file; do echo "$file" $(udevadm info -q path -n "$file"); done
 echo ==== DMESG ====
 dmesg
 echo ==== SYSLG ====
--- a/shared/Makefile
+++ b/shared/Makefile
@@ -13,14 +13,14 @@ libagoclient.so: agoclient.h agoclient.o CDataFile.o
 	test -h $(@) || ln -s $(@)$(LIB_VER) $(@)
 
 agoclient.py: agoclient.py.in
-	sed -e 's|@BINDIR|$(BINDIR)|g' \
-		-e 's|@CONFDIR|$(CONFDIR)|g' $< > $@
+	sed -e 's|@BINDIR@|$(BINDIR)|g' \
+		-e 's|@CONFDIR@|$(CONFDIR)|g' $< > $@
 
 agoclient.h: agoclient.h.in
-	sed -e 's|@BINDIR|$(BINDIR)|g' \
-		-e 's|@CONFDIR|$(CONFDIR)|g' \
-		-e 's|@LOCALSTATEDIR|$(LOCALSTATEDIR)|g' \
-		-e 's|@HTMLDIR|$(HTMLDIR)|g' $< > $@
+	sed -e 's|@BINDIR@|$(BINDIR)|g' \
+		-e 's|@CONFDIR@|$(CONFDIR)|g' \
+		-e 's|@LOCALSTATEDIR@|$(LOCALSTATEDIR)|g' \
+		-e 's|@HTMLDIR@|$(HTMLDIR)|g' $< > $@
 
 clean:
 	rm -f agoclient.h agoclient.o CDataFile.o libagoclient.so* agoclient.py
--- a/shared/agoclient.h.in
+++ b/shared/agoclient.h.in
@@ -21,10 +21,10 @@
 
 #include "CDataFile.h"
 
-#define BINDIR "@BINDIR"
-#define CONFDIR "@CONFDIR"
-#define LOCALSTATEDIR "@LOCALSTATEDIR"
-#define HTMLDIR "@HTMLDIR"
+#define BINDIR "@BINDIR@"
+#define CONFDIR "@CONFDIR@"
+#define LOCALSTATEDIR "@LOCALSTATEDIR@"
+#define HTMLDIR "@HTMLDIR@"
 
 #define CONFIG_FILE_PATH CONFDIR "/conf.d"
 
--- a/shared/agoclient.py.in
+++ b/shared/agoclient.py.in
@@ -16,8 +16,8 @@ from qpid.log import enable, DEBUG, WARN
 
 syslog.openlog(sys.argv[0], syslog.LOG_PID, syslog.LOG_DAEMON)
 
-BINDIR = '@BINDIR'
-CONFDIR = '@CONFDIR'
+BINDIR = '@BINDIR@'
+CONFDIR = '@CONFDIR@'
 
 CONFIG_LOCK = Lock()
 
